ikelani421/SmartVisitorManagementBend
# Refactor codebase to es6 & Add application logs

#### What does this PR do?
- Sets up babel for ES6+ code
- Sets up eslint for code linting
- Refactors codebase to ES6+ and change folder structure
- Sets up morgan and winston to create application log files

#### Description of Task to be completed?
- [x] Set up babel for ES6+ code
- [x] Set up eslint for code linting
- [x] Set up application logs
- [x] Refactor codebase to ES6+ and change folder structure

#### How should this be manually tested?
Clone this branch onto your local machine and do the following;
install new dependencies: `npm install`
run the new development server script: `npm run dev`
to run build for production: `npm start` will trigger a `prestart` script that builds out the code

Upon starting the server and making requests, you should have a new folder created for the application logs at `./logs/all-logs.log`

#### Any background context you want to provide?
N/a

#### What are the relevant JIRA stories?
N/a

#### Screenshots (if appropriate)
N/a

#### Questions:
TODO: Create application response module
